dashboard/app/controllers/curriculum_tracking_pixel_controller.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/firehose'
+
+class CurriculumTrackingPixelController < ApplicationController
+  def index
+    prevent_caching
+    curriculum_page = URI.unescape(params[:from])
+    user_id = current_user&.id
+
+    FirehoseClient.instance.put_record(
+      study: 'curriculum-builder-page-views',
+      study_group: 'v0',
+      event: 'curriculum-builder-page-view',
+      data_json: {
+        curriculumBuilderUrl: curriculum_page,
+        unplugged: curriculum_page.include?("unplugged"),
+        userId: user_id
+      }.to_json
+    )
+
+    send_file pegasus_dir('sites.v3/code.org/public/images/1x1.png'), type: 'image/png'
```

Comment:
nit: we could put this (or a copy) in `/shared` or in the dashboard app itself, just to keep things decoupled from pegasus.